contrib/linux/actions/rsync.yaml

```diff
@@ -9,21 +9,20 @@
             type: 'string'
             description: 'List of files/directories to to be copied'
             required: true
-            position: 1
+            position: 0
```

Comment:
Is the intent to pass the parameters to the st2 action like this?  `linux.rsync <source> <dest_server> <destination>` instead of `linux.rsync source=<source> dest_server=<dest_server> destination=<destination>`?